### ISSUE TITLE
[Bug] Crashed when creating a new world

### ISSUE THREAD

**Issue by notepower2k1** [paulevsGitch/BetterNether#500]
### What happened?

What happened?
I can play the old world , if I create a new world , Minecraft will be crashed

### BetterNether

6.0.8

### BCLib

1.2.4

### Fabric API

0.46.1

### Fabric Loader

0.12.12

### Minecraft

1.18.1

### Relevant log output

```shell
[20:18:92] [Render thread/INFO]: [DataFixerAPI] Applying Patch{betternether:5.3.5:21020677}
[20:18:92] [Render thread/INFO]: [DataFixerAPI] Applying Patch{bclib:1.2.1:4227073}
[20:18:92] [Render thread/INFO]: [DataFixerAPI] Updating Patch-Level for 'betternether' from 0.0.0 to 5.3.5
[20:18:92] [Render thread/INFO]: [DataFixerAPI] Updating Patch-Level for 'bclib' from 0.0.0 to 1.2.1
[20:18:92] [Render thread/INFO]: [betternether] Using legacy (1.17) generator: false
[20:18:23] [Render thread/INFO]: Adding serverside commands..
[20:18:31] [Render thread/WARN]: Ambiguity between arguments [teleport, location] and [teleport, destination] with inputs: [0.1 -0.5 .9, 0 0 0]
[20:18:31] [Render thread/WARN]: Ambiguity between arguments [teleport, location] and [teleport, targets] with inputs: [0.1 -0.5 .9, 0 0 0]
[20:18:32] [Render thread/WARN]: Ambiguity between arguments [teleport, destination] and [teleport, targets] with inputs: [Player, 0123, @e, dd12be42-52a9-4a91-a8a1-11c01849e498]
[20:18:32] [Render thread/WARN]: Ambiguity between arguments [teleport, targets] and [teleport, destination] with inputs: [Player, 0123, dd12be42-52a9-4a91-a8a1-11c01849e498]
[20:18:32] [Render thread/WARN]: Ambiguity between arguments [teleport, targets, location] and [teleport, targets, destination] with inputs: [0.1 -0.5 .9, 0 0 0]
[20:18:32] [Render thread/WARN]: Ambiguity between arguments [ftbchunks, admin, view_loaded_chunks, reset] and [ftbchunks, admin, view_loaded_chunks, dimension] with inputs: [reset]
[20:18:33] [Render thread/WARN]: Ambiguity between arguments [ftbteams, info, server_id] and [ftbteams, info, team] with inputs: [server_id]
[20:18:36] [Render thread/INFO]: Reloading ResourceManager: Default, Fabric Mods (AdventureZ, Amplified Nether for Fabric, Artifality, Awesome Dungeon Edition Ocean, Better Animals Plus, Better Nether, Bountiful, BYG, Campanion, Cave Tweaks, Config to Brigadier, Croptopia, Dark Enchanting, When Dungeons Arise, Dungeon vanilla, Earth2Java, Eldritch Mobs, End Goblin Traders, Enhanced Celestials, Fabric Tool Attribute API (v1), Fabric Tailor, Feyrune, Florist's Mod, Friends&Foes, FTB Chunks, More Geodes, Goblin Traders, Golems Galore, The Graveyard, Guild, Hardcore Torches, Impaled, Inmis, LibCapableData, MoreVillagersFabric, Mo' Structures, Patchouli, Promenade, Repurposed Structures, Rings of Ascension, This Rocks!, Small Artifacts, Stoneholm, Terralith for Fabric, Things, Tiny Skeletons, Towers of the Wild: Additions, Towers of the Wild: Reworked, Trinkets, Waystones)
[20:18:45] [Worker-Main-2/INFO]: [STDOUT]: endgoblintraders:trades/end_goblin_trader/common.json
[20:18:45] [Worker-Main-2/INFO]: [STDOUT]: endgoblintraders:trades/end_goblin_trader/epic.json
[20:18:45] [Worker-Main-2/INFO]: [STDOUT]: endgoblintraders:trades/end_goblin_trader/legendary.json
[20:18:45] [Worker-Main-2/INFO]: [STDOUT]: endgoblintraders:trades/end_goblin_trader/rare.json
[20:18:45] [Worker-Main-2/INFO]: [STDOUT]: endgoblintraders:trades/end_goblin_trader/uncommon.json
[20:18:52] [Worker-Main-2/INFO]: [STDOUT]: goblintraders:trades/goblin_trader/common.json
[20:18:53] [Worker-Main-2/INFO]: [STDOUT]: goblintraders:trades/goblin_trader/legendary.json
[20:18:53] [Worker-Main-2/INFO]: [STDOUT]: goblintraders:trades/goblin_trader/rare.json
[20:18:53] [Worker-Main-2/INFO]: [STDOUT]: goblintraders:trades/goblin_trader/uncommon.json
[20:18:55] [Worker-Main-2/INFO]: [STDOUT]: goblintraders:trades/vein_goblin_trader/common.json
[20:18:56] [Worker-Main-2/INFO]: [STDOUT]: goblintraders:trades/vein_goblin_trader/epic.json
[20:18:56] [Worker-Main-2/INFO]: [STDOUT]: goblintraders:trades/vein_goblin_trader/legendary.json
[20:18:56] [Worker-Main-2/INFO]: [STDOUT]: goblintraders:trades/vein_goblin_trader/rare.json
[20:18:56] [Worker-Main-2/INFO]: [STDOUT]: goblintraders:trades/vein_goblin_trader/uncommon.json
[20:18:03] [Render thread/INFO]: Loaded 7 recipes
[20:18:19] [Render thread/ERROR]: Couldn't parse loot table dungeons_arise:chests/pillager_palace_supply
com.google.gson.JsonSyntaxException: Missing type, expected to find a string
	at net.minecraft.class_3518.method_15265(class_3518.java:112)
	at net.minecraft.class_144$class_145.method_627(class_144.java:97)
	at net.minecraft.class_144$class_145.method_530(class_144.java:81)
	at net.minecraft.class_120$class_123.method_528(class_120.java:99)
	at net.minecraft.class_120$class_123.method_517(class_120.java:88)
	at net.minecraft.class_5330$class_5333.deserialize(class_5330.java:102)
	at com.google.gson.internal.bind.TreeTypeAdapter.read(TreeTypeAdapter.java:69)
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.read(TypeAdapterRuntimeTypeWrapper.java:41)
	at com.google.gson.internal.bind.ArrayTypeAdapter.read(ArrayTypeAdapter.java:72)
	at com.google.gson.Gson.fromJson(Gson.java:932)
	at com.google.gson.Gson.fromJson(Gson.java:1003)
	at com.google.gson.internal.bind.TreeTypeAdapter$GsonContextImpl.deserialize(TreeTypeAdapter.java:162)
	at net.minecraft.class_3518.method_15291(class_3518.java:448)
	at net.minecraft.class_3518.method_15283(class_3518.java:466)
	at net.minecraft.class_85$class_90.method_441(class_85.java:149)
	at net.minecraft.class_85$class_90.method_424(class_85.java:129)
	at net.minecraft.class_79$class_5337.method_29319(class_79.java:86)
	at net.minecraft.class_79$class_5337.method_517(class_79.java:74)
	at net.minecraft.class_5330$class_5333.deserialize(class_5330.java:102)
	at com.google.gson.internal.bind.TreeTypeAdapter.read(TreeTypeAdapter.java:69)
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.read(TypeAdapterRuntimeTypeWrapper.java:41)
	at com.google.gson.internal.bind.ArrayTypeAdapter.read(ArrayTypeAdapter.java:72)
	at com.google.gson.Gson.fromJson(Gson.java:932)
	at com.google.gson.Gson.fromJson(Gson.java:1003)
	at com.google.gson.internal.bind.TreeTypeAdapter$GsonContextImpl.deserialize(TreeTypeAdapter.java:162)
	at net.minecraft.class_3518.method_15291(class_3518.java:448)
	at net.minecraft.class_3518.method_15272(class_3518.java:456)
	at net.minecraft.class_55$class_57.method_358(class_55.java:173)
	at net.minecraft.class_55$class_57.deserialize(class_55.java:169)
	at com.google.gson.internal.bind.TreeTypeAdapter.read(TreeTypeAdapter.java:69)
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.read(TypeAdapterRuntimeTypeWrapper.java:41)
	at com.google.gson.internal.bind.ArrayTypeAdapter.read(ArrayTypeAdapter.java:72)
	at com.google.gson.Gson.fromJson(Gson.java:932)
	at com.google.gson.Gson.fromJson(Gson.java:1003)
	at com.google.gson.internal.bind.TreeTypeAdapter$GsonContextImpl.deserialize(TreeTypeAdapter.java:162)
	at net.minecraft.class_3518.method_15291(class_3518.java:448)
	at net.minecraft.class_3518.method_15283(class_3518.java:466)
	at net.minecraft.class_52$class_54.method_340(class_52.java:213)
	at net.minecraft.class_52$class_54.deserialize(class_52.java:209)
	at com.google.gson.internal.bind.TreeTypeAdapter.read(TreeTypeAdapter.java:69)
	at com.google.gson.Gson.fromJson(Gson.java:932)
	at com.google.gson.Gson.fromJson(Gson.java:1003)
	at com.google.gson.Gson.fromJson(Gson.java:975)
	at net.minecraft.class_60.method_20711(class_60.java:44)
	at java.base/java.util.HashMap.forEach(HashMap.java:1421)
	at net.minecraft.class_60.method_20712(class_60.java:42)
	at net.minecraft.class_60.method_18788(class_60.java:17)
	at net.minecraft.class_4080.method_18790(class_4080.java:13)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:718)
	at java.base/java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482)
	at net.minecraft.class_4014.method_18365(class_4014.java:71)
	at net.minecraft.class_1255.method_18859(class_1255.java:151)
	at net.minecraft.class_4093.method_18859(class_4093.java:23)
	at net.minecraft.class_1255.method_16075(class_1255.java:125)
	at net.minecraft.class_1255.method_18857(class_1255.java:134)
	at net.minecraft.class_310.method_29604(class_310.java:2154)
	at net.minecraft.class_310.method_29610(class_310.java:2003)
	at net.minecraft.class_310.method_29607(class_310.java:15501)
	at net.minecraft.class_525.method_2736(class_525.java:278)
	at net.minecraft.class_525.method_19922(class_525.java:214)
	at net.minecraft.class_4185.method_25306(class_4185.java:40)
	at net.minecraft.class_4264.method_25348(class_4264.java:16)
	at net.minecraft.class_339.method_25402(class_339.java:121)
	at net.minecraft.class_4069.method_25402(class_4069.java:27)
	at net.minecraft.class_312.method_1611(class_312.java:94)
	at net.minecraft.class_437.method_25412(class_437.java:491)
	at net.minecraft.class_312.method_1601(class_312.java:94)
	at net.minecraft.class_312.method_22686(class_312.java:165)
	at net.minecraft.class_1255.execute(class_1255.java:101)
	at net.minecraft.class_312.method_22684(class_312.java:165)
	at org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:36)
	at org.lwjgl.system.JNI.invokeV(Native Method)
	at org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3101)
	at com.mojang.blaze3d.systems.RenderSystem.flipFrame(RenderSystem.java:195)
	at net.minecraft.class_1041.method_15998(class_1041.java:310)
	at net.minecraft.class_310.method_1523(class_310.java:1142)
	at net.minecraft.class_310.method_1514(class_310.java:733)
	at net.minecraft.client.main.Main.main(Main.java:238)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:608)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

[20:18:19] [Render thread/ERROR]: Couldn't parse loot table dungeons_arise:chests/pillager_palace_treasure
com.google.gson.JsonSyntaxException: Missing type, expected to find a string
	at net.minecraft.class_3518.method_15265(class_3518.java:112)
	at net.minecraft.class_144$class_145.method_627(class_144.java:97)
	at net.minecraft.class_144$class_145.method_530(class_144.java:81)
	at net.minecraft.class_120$class_123.method_528(class_120.java:99)
	at net.minecraft.class_120$class_123.method_517(class_120.java:88)
	at net.minecraft.class_5330$class_5333.deserialize(class_5330.java:102)
	at com.google.gson.internal.bind.TreeTypeAdapter.read(TreeTypeAdapter.java:69)
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.read(TypeAdapterRuntimeTypeWrapper.java:41)
	at com.google.gson.internal.bind.ArrayTypeAdapter.read(ArrayTypeAdapter.java:72)
	at com.google.gson.Gson.fromJson(Gson.java:932)
	at com.google.gson.Gson.fromJson(Gson.java:1003)
	at com.google.gson.internal.bind.TreeTypeAdapter$GsonContextImpl.deserialize(TreeTypeAdapter.java:162)
	at net.minecraft.class_3518.method_15291(class_3518.java:448)
	at net.minecraft.class_3518.method_15283(class_3518.java:466)
	at net.minecraft.class_85$class_90.method_441(class_85.java:149)
	at net.minecraft.class_85$class_90.method_424(class_85.java:129)
	at net.minecraft.class_79$class_5337.method_29319(class_79.java:86)
	at net.minecraft.class_79$class_5337.method_517(class_79.java:74)
	at net.minecraft.class_5330$class_5333.deserialize(class_5330.java:102)
	at com.google.gson.internal.bind.TreeTypeAdapter.read(TreeTypeAdapter.java:69)
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.read(TypeAdapterRuntimeTypeWrapper.java:41)
	at com.google.gson.internal.bind.ArrayTypeAdapter.read(ArrayTypeAdapter.java:72)
	at com.google.gson.Gson.fromJson(Gson.java:932)
	at com.google.gson.Gson.fromJson(Gson.java:1003)
	at com.google.gson.internal.bind.TreeTypeAdapter$GsonContextImpl.deserialize(TreeTypeAdapter.java:162)
	at net.minecraft.class_3518.method_15291(class_3518.java:448)
	at net.minecraft.class_3518.method_15272(class_3518.java:456)
	at net.minecraft.class_55$class_57.method_358(class_55.java:173)
	at net.minecraft.class_55$class_57.deserialize(class_55.java:169)
	at com.google.gson.internal.bind.TreeTypeAdapter.read(TreeTypeAdapter.java:69)
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.read(TypeAdapterRuntimeTypeWrapper.java:41)
	at com.google.gson.internal.bind.ArrayTypeAdapter.read(ArrayTypeAdapter.java:72)
	at com.google.gson.Gson.fromJson(Gson.java:932)
	at com.google.gson.Gson.fromJson(Gson.java:1003)
	at com.google.gson.internal.bind.TreeTypeAdapter$GsonContextImpl.deserialize(TreeTypeAdapter.java:162)
	at net.minecraft.class_3518.method_15291(class_3518.java:448)
	at net.minecraft.class_3518.method_15283(class_3518.java:466)
	at net.minecraft.class_52$class_54.method_340(class_52.java:213)
	at net.minecraft.class_52$class_54.deserialize(class_52.java:209)
	at com.google.gson.internal.bind.TreeTypeAdapter.read(TreeTypeAdapter.java:69)
	at com.google.gson.Gson.fromJson(Gson.java:932)
	at com.google.gson.Gson.fromJson(Gson.java:1003)
	at com.google.gson.Gson.fromJson(Gson.java:975)
	at net.minecraft.class_60.method_20711(class_60.java:44)
	at java.base/java.util.HashMap.forEach(HashMap.java:1421)
	at net.minecraft.class_60.method_20712(class_60.java:42)
	at net.minecraft.class_60.method_18788(class_60.java:17)
	at net.minecraft.class_4080.method_18790(class_4080.java:13)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:718)
	at java.base/java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482)
	at net.minecraft.class_4014.method_18365(class_4014.java:71)
	at net.minecraft.class_1255.method_18859(class_1255.java:151)
	at net.minecraft.class_4093.method_18859(class_4093.java:23)
	at net.minecraft.class_1255.method_16075(class_1255.java:125)
	at net.minecraft.class_1255.method_18857(class_1255.java:134)
	at net.minecraft.class_310.method_29604(class_310.java:2154)
	at net.minecraft.class_310.method_29610(class_310.java:2003)
	at net.minecraft.class_310.method_29607(class_310.java:15501)
	at net.minecraft.class_525.method_2736(class_525.java:278)
	at net.minecraft.class_525.method_19922(class_525.java:214)
	at net.minecraft.class_4185.method_25306(class_4185.java:40)
	at net.minecraft.class_4264.method_25348(class_4264.java:16)
	at net.minecraft.class_339.method_25402(class_339.java:121)
	at net.minecraft.class_4069.method_25402(class_4069.java:27)
	at net.minecraft.class_312.method_1611(class_312.java:94)
	at net.minecraft.class_437.method_25412(class_437.java:491)
	at net.minecraft.class_312.method_1601(class_312.java:94)
	at net.minecraft.class_312.method_22686(class_312.java:165)
	at net.minecraft.class_1255.execute(class_1255.java:101)
	at net.minecraft.class_312.method_22684(class_312.java:165)
	at org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:36)
	at org.lwjgl.system.JNI.invokeV(Native Method)
	at org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3101)
	at com.mojang.blaze3d.systems.RenderSystem.flipFrame(RenderSystem.java:195)
	at net.minecraft.class_1041.method_15998(class_1041.java:310)
	at net.minecraft.class_310.method_1523(class_310.java:1142)
	at net.minecraft.class_310.method_1514(class_310.java:733)
	at net.minecraft.client.main.Main.main(Main.java:238)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:608)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

[20:18:34] [Render thread/ERROR]: Couldn't parse loot table bountiful:blocks/bountyboard
com.google.gson.JsonSyntaxException: Missing type, expected to find a string
	at net.minecraft.class_3518.method_15265(class_3518.java:112)
	at net.minecraft.class_134$class_136.method_605(class_134.java:112)
	at net.minecraft.class_134$class_136.method_530(class_134.java:101)
	at net.minecraft.class_120$class_123.method_528(class_120.java:99)
	at net.minecraft.class_120$class_123.method_517(class_120.java:88)
	at net.minecraft.class_5330$class_5333.deserialize(class_5330.java:102)
	at com.google.gson.internal.bind.TreeTypeAdapter.read(TreeTypeAdapter.java:69)
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.read(TypeAdapterRuntimeTypeWrapper.java:41)
	at com.google.gson.internal.bind.ArrayTypeAdapter.read(ArrayTypeAdapter.java:72)
	at com.google.gson.Gson.fromJson(Gson.java:932)
	at com.google.gson.Gson.fromJson(Gson.java:1003)
	at com.google.gson.internal.bind.TreeTypeAdapter$GsonContextImpl.deserialize(TreeTypeAdapter.java:162)
	at net.minecraft.class_3518.method_15291(class_3518.java:448)
	at net.minecraft.class_3518.method_15283(class_3518.java:466)
	at net.minecraft.class_85$class_90.method_441(class_85.java:149)
	at net.minecraft.class_85$class_90.method_424(class_85.java:129)
	at net.minecraft.class_79$class_5337.method_29319(class_79.java:86)
	at net.minecraft.class_79$class_5337.method_517(class_79.java:74)
	at net.minecraft.class_5330$class_5333.deserialize(class_5330.java:102)
	at com.google.gson.internal.bind.TreeTypeAdapter.read(TreeTypeAdapter.java:69)
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.read(TypeAdapterRuntimeTypeWrapper.java:41)
	at com.google.gson.internal.bind.ArrayTypeAdapter.read(ArrayTypeAdapter.java:72)
	at com.google.gson.Gson.fromJson(Gson.java:932)
	at com.google.gson.Gson.fromJson(Gson.java:1003)
	at com.google.gson.internal.bind.TreeTypeAdapter$GsonContextImpl.deserialize(TreeTypeAdapter.java:162)
	at net.minecraft.class_3518.method_15291(class_3518.java:448)
	at net.minecraft.class_3518.method_15272(class_3518.java:456)
	at net.minecraft.class_55$class_57.method_358(class_55.java:173)
	at net.minecraft.class_55$class_57.deserialize(class_55.java:169)
	at com.google.gson.internal.bind.TreeTypeAdapter.read(TreeTypeAdapter.java:69)
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.read(TypeAdapterRuntimeTypeWrapper.java:41)
	at com.google.gson.internal.bind.ArrayTypeAdapter.read(ArrayTypeAdapter.java:72)
	at com.google.gson.Gson.fromJson(Gson.java:932)
	at com.google.gson.Gson.fromJson(Gson.java:1003)
	at com.google.gson.internal.bind.TreeTypeAdapter$GsonContextImpl.deserialize(TreeTypeAdapter.java:162)
	at net.minecraft.class_3518.method_15291(class_3518.java:448)
	at net.minecraft.class_3518.method_15283(class_3518.java:466)
	at net.minecraft.class_52$class_54.method_340(class_52.java:213)
	at net.minecraft.class_52$class_54.deserialize(class_52.java:209)
	at com.google.gson.internal.bind.TreeTypeAdapter.read(TreeTypeAdapter.java:69)
	at com.google.gson.Gson.fromJson(Gson.java:932)
	at com.google.gson.Gson.fromJson(Gson.java:1003)
	at com.google.gson.Gson.fromJson(Gson.java:975)
	at net.minecraft.class_60.method_20711(class_60.java:44)
	at java.base/java.util.HashMap.forEach(HashMap.java:1421)
	at net.minecraft.class_60.method_20712(class_60.java:42)
	at net.minecraft.class_60.method_18788(class_60.java:17)
	at net.minecraft.class_4080.method_18790(class_4080.java:13)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:718)
	at java.base/java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482)
	at net.minecraft.class_4014.method_18365(class_4014.java:71)
	at net.minecraft.class_1255.method_18859(class_1255.java:151)
	at net.minecraft.class_4093.method_18859(class_4093.java:23)
	at net.minecraft.class_1255.method_16075(class_1255.java:125)
	at net.minecraft.class_1255.method_18857(class_1255.java:134)
	at net.minecraft.class_310.method_29604(class_310.java:2154)
	at net.minecraft.class_310.method_29610(class_310.java:2003)
	at net.minecraft.class_310.method_29607(class_310.java:15501)
	at net.minecraft.class_525.method_2736(class_525.java:278)
	at net.minecraft.class_525.method_19922(class_525.java:214)
	at net.minecraft.class_4185.method_25306(class_4185.java:40)
	at net.minecraft.class_4264.method_25348(class_4264.java:16)
	at net.minecraft.class_339.method_25402(class_339.java:121)
	at net.minecraft.class_4069.method_25402(class_4069.java:27)
	at net.minecraft.class_312.method_1611(class_312.java:94)
	at net.minecraft.class_437.method_25412(class_437.java:491)
	at net.minecraft.class_312.method_1601(class_312.java:94)
	at net.minecraft.class_312.method_22686(class_312.java:165)
	at net.minecraft.class_1255.execute(class_1255.java:101)
	at net.minecraft.class_312.method_22684(class_312.java:165)
	at org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:36)
	at org.lwjgl.system.JNI.invokeV(Native Method)
	at org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3101)
	at com.mojang.blaze3d.systems.RenderSystem.flipFrame(RenderSystem.java:195)
	at net.minecraft.class_1041.method_15998(class_1041.java:310)
	at net.minecraft.class_310.method_1523(class_310.java:1142)
	at net.minecraft.class_310.method_1514(class_310.java:733)
	at net.minecraft.client.main.Main.main(Main.java:238)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:608)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

[20:18:56] [Render thread/ERROR]: Parsing error loading custom advancement floristmod:recipes/misc/pink_dye_from_bubblegumbush: Unknown item id 'floristmod:bubblegum_bush'
[20:18:64] [Render thread/INFO]: Loaded 1537 advancements
[20:18:26] [Render thread/INFO]: Applied 7828 biome modifications to 196 of 196 new biomes in 5.603 s
[20:18:68] [Render thread/INFO]: Registered biome provider datapack:biome_provider to index 3
[20:18:68] [Render thread/INFO]: Adding overworld biomes for datapack datapack:biome_provider with uniqueness 3
[20:18:89] [Render thread/INFO]: Merged generation settings with datapack
[20:18:69] [Render thread/FATAL]: Reported exception thrown!
net.minecraft.class_148: Starting integrated server
	at net.minecraft.class_437.method_25412(class_437.java:496)
	at net.minecraft.class_312.method_1601(class_312.java:94)
	at net.minecraft.class_312.method_22686(class_312.java:165)
	at net.minecraft.class_1255.execute(class_1255.java:101)
	at net.minecraft.class_312.method_22684(class_312.java:165)
	at org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:36)
	at org.lwjgl.system.JNI.invokeV(Native Method)
	at org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3101)
	at com.mojang.blaze3d.systems.RenderSystem.flipFrame(RenderSystem.java:195)
	at net.minecraft.class_1041.method_15998(class_1041.java:310)
	at net.minecraft.class_310.method_1523(class_310.java:1142)
	at net.minecraft.class_310.method_1514(class_310.java:733)
	at net.minecraft.client.main.Main.main(Main.java:238)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:608)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: java.lang.StackOverflowError
	at com.mojang.serialization.codecs.KeyDispatchCodec.getCodec(KeyDispatchCodec.java:123)
	at com.mojang.serialization.codecs.KeyDispatchCodec.lambda$new$0(KeyDispatchCodec.java:46)
	at com.mojang.serialization.codecs.KeyDispatchCodec.encode(KeyDispatchCodec.java:79)
	at com.mojang.serialization.codecs.KeyDispatchCodec.encode(KeyDispatchCodec.java:92)
	at com.mojang.serialization.codecs.KeyDispatchCodec.encode(KeyDispatchCodec.java:92)
	at com.mojang.serialization.codecs.KeyDispatchCodec.encode(KeyDispatchCodec.java:92)
	at com.mojang.serialization.codecs.KeyDispatchCodec.encode(KeyDispatchCodec.java:92)
	at com.mojang.serialization.codecs.KeyDispatchCodec.encode(KeyDispatchCodec.java:92)
	at com.mojang.serialization.codecs.KeyDispatchCodec.encode(KeyDispatchCodec.java:92)
	at com.mojang.serialization.codecs.KeyDispatchCodec.encode(KeyDispatchCodec.java:92)
	at com.mojang.serialization.codecs.KeyDispatchCodec.encode(KeyDispatchCodec.java:92)

-- Starting integrated server --
Details:
	Level ID: New World (16)
	Level Name: New World
Stacktrace:
	at net.minecraft.class_310.method_29610(class_310.java:2056)
	at net.minecraft.class_310.method_29607(class_310.java:15501)
	at net.minecraft.class_525.method_2736(class_525.java:278)
	at net.minecraft.class_525.method_19922(class_525.java:214)
	at net.minecraft.class_4185.method_25306(class_4185.java:40)
	at net.minecraft.class_4264.method_25348(class_4264.java:16)
	at net.minecraft.class_339.method_25402(class_339.java:121)
	at net.minecraft.class_4069.method_25402(class_4069.java:27)
	at net.minecraft.class_312.method_1611(class_312.java:94)
	at net.minecraft.class_437.method_25412(class_437.java:491)
	at net.minecraft.class_312.method_1601(class_312.java:94)
	at net.minecraft.class_312.method_22686(class_312.java:165)
	at net.minecraft.class_1255.execute(class_1255.java:101)
	at net.minecraft.class_312.method_22684(class_312.java:165)
	at org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:36)
	at org.lwjgl.system.JNI.invokeV(Native Method)
	at org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3101)
	at com.mojang.blaze3d.systems.RenderSystem.flipFrame(RenderSystem.java:195)
	at net.minecraft.class_1041.method_15998(class_1041.java:310)
	at net.minecraft.class_310.method_1523(class_310.java:1142)
	at net.minecraft.class_310.method_1514(class_310.java:733)
	at net.minecraft.client.main.Main.main(Main.java:238)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:608)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- Affected screen --
Details:
	Screen name: net.minecraft.class_525

-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: Default, Fabric Mods, illuminations/lowerres, illuminations/pixelaccurate, artifality/fancyclusters

-- System Details --
Details:
	Minecraft Version: 1.18.1
	Minecraft Version ID: 1.18.1
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 17.0.1, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 2194530456 bytes (2092 MiB) / 2852126720 bytes (2720 MiB) up to 3154116608 bytes (3008 MiB)
	CPUs: 8
	Processor Vendor: GenuineIntel
	Processor Name: 11th Gen Intel(R) Core(TM) i5-1135G7 @ 2.40GHz
	Identifier: Intel64 Family 6 Model 140 Stepping 1
	Microarchitecture: Tiger Lake
	Frequency (GHz): 2.42
	Number of physical packages: 1
	Number of physical CPUs: 4
	Number of logical CPUs: 8
	Graphics card #0 name: Intel(R) Iris(R) Xe Graphics
	Graphics card #0 vendor: Intel Corporation (0x8086)
	Graphics card #0 VRAM (MB): 1024.00
	Graphics card #0 deviceId: 0x9a49
	Graphics card #0 versionInfo: DriverVersion=30.0.100.9955
	Graphics card #1 name: NVIDIA GeForce MX450
	Graphics card #1 vendor: NVIDIA (0x10de)
	Graphics card #1 VRAM (MB): 2048.00
	Graphics card #1 deviceId: 0x1f97
	Graphics card #1 versionInfo: DriverVersion=27.21.14.6280
	Memory slot #0 capacity (MB): 8192.00
	Memory slot #0 clockSpeed (GHz): 3.20
	Memory slot #0 type: DDR4
	Virtual memory max (MB): 20245.99
	Virtual memory used (MB): 12108.05
	Swap memory total (MB): 12288.00
	Swap memory used (MB): 1122.57
	JVM Flags: 10 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmn128M -Xmx3000M -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
```

### Other Mods

```shell
Fabric Mods: 
		adventurez: AdventureZ 1.4.7
		amplified_nether: Amplified Nether for Fabric 1.0.1
		antighost: AntiGhost 1.18-fabric0.43.1-1.1.4
		architectury: Architectury 3.4.9
		artifality: Artifality 0.5.4
		awesomedungeon_ocean: Awesome Dungeon Edition Ocean 1.0.3
		bclib: BCLib 1.2.4
		betteranimalsplus: Better Animals Plus 1.18.1-11.0.2
		betterdroppeditems: Better Dropped Items 1.3.1-21w37a
		betternether: Better Nether 6.0.8
		bettersleeping: BetterSleeping 0.5.1+1.18
		betterthirdperson: Better Third Person 1.6.0
		blue_endless_jankson: jankson 1.2.1
		blur: Blur (Fabric) 2.4.1
		bountiful: Bountiful 2.0.1
		byg: BYG 1.3.5.2
		campanion: Campanion 3.0.1
		capybara: Capybara 1.0.1
		cardinal-components-base: Cardinal Components API (base) 4.0.1
		cardinal-components-entity: Cardinal Components API (entities) 4.0.1
		cardinal-components-world: Cardinal Components API (worlds) 4.0.1
		cavetweaks: Cave Tweaks 1.0
		cleancut: CleanCut 1.18-4.0-fabric
		cloth-basic-math: cloth-basic-math 0.6.0
		cloth-config: Cloth Config v6 6.1.48
		com_electronwill_night-config_core: core 3.6.4
		com_electronwill_night-config_toml: toml 3.6.4
		com_eliotlash_mclib_mclib: mclib 18
		com_eliotlash_molang_molang: molang 18
		com_moandjiezana_toml_toml4j: toml4j 0.7.2
		com_typesafe_config: config 1.4.1
		com_velocitypowered_velocity-native: velocity-native 1.1.0-SNAPSHOT
		confabricate: confabricate 2.2.0-SNAPSHOT+4.1.1
		config2brigadier: Config to Brigadier 1.0.2
		cosmetica: Cosmetica 0.3.2
		croptopia: Croptopia 1.7.4
		crowdin-translate: CrowdinTranslate 1.4+1.18
		dark-enchanting: Dark Enchanting 0.6.3+1.18
		dawn: Dawn API 3.2.0
		difficultyplus: Difficulty Plus 1.0.1
		dungeons_arise: When Dungeons Arise 2.1.49c-fabric
		dungeonvanilla: Dungeon vanilla 1.0.0
		dynamicfps: Dynamic FPS 2.1.0
		earthtojavamobs: Earth2Java 1.7.4+1.18
		effective: Effective 1.1
		eldritch_mobs: Eldritch Mobs 1.10.2
		endgoblintraders: End Goblin Traders 1.4.2
		enhancedcelestials: Enhanced Celestials 2.0.6
		fabric: Fabric API 0.46.1+1.18
		fabric-api-base: Fabric API Base 0.4.1+b4f4f6cd14
		fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.5.0+17be577f65
		fabric-biome-api-v1: Fabric Biome API (v1) 6.0.1+ded849a965
		fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.9+3ac43d9565
		fabric-command-api-v1: Fabric Command API (v1) 1.1.6+3ac43d9565
		fabric-commands-v0: Fabric Commands (v0) 0.2.5+b4f4f6cd65
		fabric-containers-v0: Fabric Containers (v0) 0.1.18+d154e2c665
		fabric-content-registries-v0: Fabric Content Registries (v0) 0.4.5+6f53a73d65
		fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.8+3ac43d9565
		fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.9+a1d9bbf565
		fabric-entity-events-v1: Fabric Entity Events (v1) 1.4.5+6b21378a65
		fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.16+bfa23f1765
		fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.6+b4f4f6cd65
		fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.10+3ac43d9565
		fabric-item-api-v1: Fabric Item API (v1) 1.3.0+691a79b565
		fabric-item-groups-v0: Fabric Item Groups (v0) 0.3.3+3ac43d9565
		fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.8+c8aba2f365
		fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.6+b4f4f6cd65
		fabric-language-kotlin: Fabric Language Kotlin 1.7.1+kotlin.1.6.10
		fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 1.4.10+c15ca33514
		fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.0.8+3ac43d9565
		fabric-mining-level-api-v1: Fabric Mining Level API (v1) 1.0.3+3ac43d9565
		fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.8+b4f4f6cd65
		fabric-models-v0: Fabric Models (v0) 0.3.3+3ac43d9565
		fabric-networking-api-v1: Fabric Networking API (v1) 1.0.18+3ac43d9565
		fabric-networking-v0: Fabric Networking (v0) 0.3.5+b4f4f6cd65
		fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.11.1+3b82842e65
		fabric-object-builders-v0: Fabric Object Builders (v0) 0.7.9+3ac43d9565
		fabric-particles-v1: Fabric Particles (v1) 0.2.9+526dc1ac65
		fabric-permissions-api-v0: fabric-permissions-api 0.1-SNAPSHOT
		fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.9.0+befed49e65
		fabric-renderer-api-v1: Fabric Renderer API (v1) 0.4.10+b0b66fc365
		fabric-renderer-indigo: Fabric Renderer - Indigo 0.4.13+b0b66fc365
		fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.7+b4f4f6cd65
		fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.4+7242e9d765
		fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 0.1.18+3ac43d9565
		fabric-rendering-v0: Fabric Rendering (v0) 1.1.9+b4f4f6cd65
		fabric-rendering-v1: Fabric Rendering (v1) 1.10.3+6b21378a65
		fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 1.0.1+13eda06565
		fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.4.11+3ac43d9565
		fabric-screen-api-v1: Fabric Screen API (v1) 1.0.7+3ac43d9565
		fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.1.11+3ac43d9565
		fabric-structure-api-v1: Fabric Structure API (v1) 2.1.0+0a73a6c265
		fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 1.2.5+3ac43d9565
		fabric-textures-v0: Fabric Textures (v0) 1.0.9+3ac43d9565
		fabric-tool-attribute-api-v1: Fabric Tool Attribute API (v1) 1.3.5+fb3b57b465
		fabric-transfer-api-v1: Fabric Transfer API (v1) 1.5.6+b4f4f6cd65
		fabricenchantments: Fabric Enchantments 0.6.1
		fabricloader: Fabric Loader 0.12.12
		fabrictailor: Fabric Tailor 1.8.0
		fallingleaves: Falling Leaves 1.9.0+1.18.1
		fallingtree: FallingTree 3.1.0
		fatxporbs: Fat Experience Orbs 2.0.0
		ferritecore: FerriteCore 4.0.0
		feyrune: Feyrune 1.18.1-0.0.1
		fiber: fiber 0.23.0-2
		floristmod: Florist's Mod fabric-1.18.1-v-2.0.0
		forgeconfigapiport: Forge Config API Port 3.1.1
		friendsandfoes: Friends&Foes 1.0.2
		ftbchunks: FTB Chunks 1801.3.3-build.113
		ftblibrary: FTB Library 1801.3.5-build.109
		ftbteams: FTB Teams 1801.2.4-build.46
		geckolib3: Geckolib 3.0.27
		geodes: More Geodes 1.2.0
		goblintraders: Goblin Traders 1.4.1
		golemsgalore: Golems Galore 1.1.1-1.18
		goosik-config: goosik-config 0.2.0
		graveyard: The Graveyard 1.3
		guild: Guild 0.3.8
		hardcore_torches: Hardcore Torches 1.2.1
		illuminations: Illuminations 1.10.2
		impaled: Impaled 1.0.2
		incantationem: Incantationem 1.1.3+1.18.1
		inmis: Inmis 2.4.0-1.18
		io_leangen_geantyref_geantyref: geantyref 1.3.11
		iris: Iris 1.2.0-pre
		jankson: Jankson 4.0.0+j1.2.0
		java: OpenJDK 64-Bit Server VM 17
		kambrik: Kambrik 3.0.1-1.18
		kirin: Kirin UI 1.10.0-beta.2
		krypton: Krypton 0.1.6
		lambdynlights: LambDynamicLights 2.1.0+1.17
		lazydfu: LazyDFU 0.1.2
		libcd: LibCapableData 3.0.3+1.16.3
		libgui: LibGui 5.2.0+1.18
		libninepatch: LibNinePatch 1.1.0
		lithium: Lithium 0.7.7
		logprot: Logprot Mod 1.18-1.4
		medievalentity: Mediaval Entity 1.0.5
		megane: megane 7.0.1
		megane-api: megane-api 7.0.0
		megane-fabric-transfer: megane-fabric-transfer 7.0.0
		megane-runtime: megane-runtime 7.0.0
		megane-vanilla: megane-vanilla 7.0.0
		midnightlib: MidnightLib 0.3.1
		minecraft: Minecraft 1.18.1
		modmenu: Mod Menu 3.0.1
		morevillagers-fabric: MoreVillagersFabric 3.1.1-SNAPSHOT
		mostructures: Mo' Structures 1.3.0-pre1+1.18
		myron: Myron 1.6.2+1.18.1
		mythic-mounts: Mythic Mounts 1.18-4.0
		no_fog: No Fog 1.2.0+1.16.5-1.18.1
		omega-config: OmegaConfig 1.2.2-1.18.1
		org_anarres_jcpp: jcpp 1.4.14
		org_apache_commons_commons-math3: commons-math3 3.3
		org_apache_commons_commons-text: commons-text 1.9
		org_aperlambda_lambdajcommon: lambdajcommon 1.8.1
		org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.6.10
		org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.6.10
		org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.6.10
		org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.6.10
		org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.5.2
		org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.5.2
		org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm: kotlinx-serialization-cbor-jvm 1.3.1
		org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.3.1
		org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.3.1
		org_joml_joml: joml 1.10.2
		org_slf4j_slf4j-api: slf4j-api 1.7.12
		org_spongepowered_configurate-core: configurate-core 4.1.1
		org_spongepowered_configurate-extra-dfu4: configurate-extra-dfu4 4.1.1
		org_spongepowered_configurate-gson: configurate-gson 4.1.1
		org_spongepowered_configurate-hocon: configurate-hocon 4.1.1
		oro_config: OroArmor Config 2.1.0
		owo: oωo 0.3.14+1.18
		patchouli: Patchouli 1.18.1-64-FABRIC
		pickupnotifier: Pick Up Notifier 3.1.0
		playerabilitylib: Pal 1.5.0
		presencefootsteps: Presence Footsteps 1.4.0
		pride: Pride Lib 1.1.0+1.17
		promenade: Promenade 2.4.0
		puzzleslib: Puzzles Lib 3.1.3
		reach-entity-attributes: Reach Entity Attributes 2.1.1
		repurposed_structures: Repurposed Structures 3.3.0+1.18.1
		ringsofascension: Rings of Ascension 1.0
		rocks: This Rocks! 1.5.2
		roughlyenoughitems: Roughly Enough Items 7.1.390
		satin: Satin 1.7.0
		seedfix: Seed Fix 1.0.0
		small_artifacts: Small Artifacts 0.1.0-1.18
		sodium: Sodium 0.4.0-alpha6+build.14
		spoornbountymobs: SpoornBountyMobs 2.2
		spruceui: SpruceUI 3.3.0+1.17
		starlight: Starlight 1.0.1+fabric.0583973
		stoneholm: Stoneholm 1.4.1
		structurized: Structurized 1.4.0+1.18
		talking-villagers: Talking Villagers 1.1.3
		terrablender: TerraBlender 1.1.0.70
		terraform-shapes-api-v1: Terraform Shapes API (v1) 2.1.0
		terralith: Terralith for Fabric 2.0.12
		things: Things 0.2.12+1.18
		thonkutil: ThonkUtil 1.3
		tinyskeletons: Tiny Skeletons 3.1.0
		torohealth: ToroHealth Damage Indicators 1.18-fabric-2
		totw_additions: Towers of the Wild: Additions 1.1.0
		towers_of_the_wild_reworked: Towers of the Wild: Reworked 2.1.0
		trinkets: Trinkets 3.1.0
		unvotedandshelved: Unvoted & Shelved 1.0.4
		villagernames: Villager Names 2.0.2
		visuality: Visuality 0.4.0
		visualworkbench: Visual Workbench 3.1.0
		waystones: Waystones 2.4.1
		wthit: wthit 4.4.1
```

**Comment by quiqueck** [comment]
Thanks for reporting, however the log would suggest that this error is cause by `dungeons_arise`, but I may be missing something. Why do you think it is related to betternether?

**Comment by spoorn** [comment]
@quiqueck I'm thinking this is caused by some incompatibility somewhere as I'm having the exact same error but only after updating all my mods in my modpack.  Removing BetterNether and BetterEnd gets rid of the crash, though it was working before I updated other mods.  Currently trying to find if it's conflicting with something

**Comment by spoorn** [comment]
Found out it conflicts with TerraBlender.  Removing TerraBlender stopped the crashing

**Comment by spoorn** [comment]
oh I see there is already an Issue created on the TerraBlender side: https://github.com/Glitchfiend/TerraBlender/issues/22

**Comment by quiqueck** [comment]
Duplicate of #503

**Comment by quiqueck** [comment]
Thanks for getting back at us, I'll close this issue as a duplicate, but you can continue to track our progress in the other thread.
